README.md

```diff
@@ -2,39 +2,42 @@
 
 A mobile optimized sliding navigation plugin.
 
-## Dependencies
+[![NPM](https://nodei.co/npm/navitron.png?downloads=true)](https://nodei.co/npm/navitron/)
 
-* [Zepto](http://zeptojs.com/)
-* [Mobify's fork of Velocity.js](http://github.com/mobify/velocity)
-* [Plugin](http://github.com/mobify/plugin)
+## Demo
 
+You can find a simple demo inside the `examples` folder in the repo. Run `grunt examples` to see it in Chrome (mobile device emulation is recommended).
 
-### Velocity
+## Dependencies
 
-If you are using Zepto, you need to load `bower_components/mobify-velocity/velocity.js` (this file comes with a jQuery shim bundled directly in it). If you are using jQuery, you need to load `bower_components/velocity/jquery.velocity.js`.
+* [jQuery](http://jquery.com/)
+* [Velocity.js](http://julian.com/research/velocity/)
+* [Plugin](https://github.com/mobify/plugin)
 
-### jQuery Support
+### Zepto Support
 
-Navitron supports jQuery but is not actively developed for it. You should be able to use Navitron directly with jQuery 2.0. While we don't actively support jQuery for Navitron, we welcome any and all issues and PRs to help us make it work.
+While we don't actively support Zepto for Navitron, we welcome any and all issues and PRs to help us make it work.
 
 ## Installation
 
-Navitron can be installed using bower:
+Navitron can be installed using NPM:
 
 ```
-bower install navitron
+npm install navitron
 ```
 
 ## Usage with Require.js
 
-We highly recommend using Require.js with Navitron. To use Require, you have to reference Navitron inside your require config file:
+We highly recommend using Require.js with Navitron. To use Require, you have to reference Navitron inside your require config file (**Note**: If your project already has these external dependencies, and the versions are compatible, it's recommended that you use the one in your project to reduce duplication):
 
 ```config.js
 
 {
     'paths': {
-        'plugin': 'bower_components/plugin/dist/plugin.min',
-        'navitron': 'bower_components/navitron/dist/navitron.min'
+        '$': 'node_modules/navitron/node_modules/jquery/dist/jquery.min',
```

Comment:
Potentially not??

This example is included for a demo of how including these dependencies _could_ work for your require.js setup. It isn't a source of truth and the preamble for it mentions that if the project already has compatible versions of the required deps to use those instead of the ones packaged with Navitron to avoid duplication.